src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddExpressionLanguageProvidersPass.php

```diff
@@ -36,10 +36,10 @@ public function process(ContainerBuilder $container)
         }
 
         // security
-        if ($container->has('security.access.expression_voter')) {
-            $definition = $container->findDefinition('security.access.expression_voter');
+        if ($container->has('security.expression_language')) {
+            $definition = $container->findDefinition('security.expression_language');
             foreach ($container->findTaggedServiceIds('security.expression_language_provider', true) as $id => $attributes) {
-                $definition->addMethodCall('addExpressionLanguageProvider', array(new Reference($id)));
+                $definition->addMethodCall('registerProvider', array(new Reference($id)));
```

Comment:
To me this makes more sense than to "take a detour" via the ExpressionVoter to extend the ExpressionLanguage service.

Should not break BC?